utility/BUILD.bazel

```diff
@@ -16,25 +17,86 @@ kt_android_library(
         "//firebase:crashlytics_deps",
         artifact("com.google.firebase:firebase-analytics:17.4.4"),
         artifact("com.google.firebase:firebase-crashlytics:17.1.1"),
-        artifact("org.jetbrains.kotlinx:kotlinx-coroutines-core"),
         artifact("org.jetbrains.kotlinx:kotlinx-coroutines-android"),
         artifact("org.jetbrains.kotlinx:kotlinx-coroutines-test"),
         artifact("com.google.truth:truth"),
         artifact("androidx.appcompat:appcompat"),
         artifact("com.github.bumptech.glide:glide"),
-        artifact("com.github.bumptech.glide:compiler"),
         artifact("com.caverock:androidsvg-aar"),
         artifact("androidx.lifecycle:lifecycle-livedata-ktx"),
-        artifact("com.crashlytics.sdk.android:crashlytics:2.9.8"),
-        artifact("io.fabric.sdk.android:fabric:1.4.7"),
-        artifact("com.google.gms:google-services:4.3.3"),
-        artifact("com.android.support:support-annotations"),
+    ],
+    visibility = ["//visibility:public"]
+)
+
+kt_android_library(
+    name = "utility_test_lib",
+    custom_package = "org.oppia.util",
+    srcs = glob(["src/test/java/org/oppia/util/**/*.kt"],),
+    resource_files = glob(["src/main/res/**/*.xml"]),
+    manifest = "src/test/TestManifest.xml",
+    deps = [
+        ":dagger",
+        ":utility_lib",
+        "//testing:testing_lib",
+        "//model:model_lib",
+        "//firebase:crashlytics_lib",
+        "//firebase:crashlytics_deps",
+        artifact("com.google.firebase:firebase-analytics:17.4.4"),
+        artifact("com.google.firebase:firebase-crashlytics:17.1.1"),
+        artifact("org.jetbrains.kotlin:kotlin-test-junit"),
+        artifact("com.google.truth:truth"),
         artifact("androidx.test.ext:junit"),
-        artifact("org.mockito:mockito-core:2.19.0"),
+        artifact("org.mockito:mockito-core"),
     ],
     visibility = ["//visibility:public"]
 )
 
-#TODO: Get android_local_test working with kotlin files
+utility_test(
+   name = "AsyncResultTest",
+    src = ["src/test/java/org/oppia/util/data/AsyncResultTest.kt"],
```

Comment:
Good catch! Thanks!!

Reply:
It will be good to have some consistency here with all other tests? All the indentations should be the same. 
Currently, it looks like this:
```
utility_test(
   name = "AsyncResultTest",
   src = ["src/test/java/org/oppia/util/data/AsyncResultTest.kt"],
   test_class = "org.oppia.util.data.AsyncResultTest",
 )

utility_test(
    name = "DataProvidersTest",
    src = ["src/test/java/org/oppia/util/data/DataProvidersTest.kt"],
    test_class = "org.oppia.util.data.DataProvidersTest",
)
```

Reply:
Fixed!